lib/command.js

```diff
@@ -445,6 +446,22 @@ const commands = {
 						});
 				});
 		});
+	},
+
+	checkout(event, statusBar){
+		return getFiles(event.target).then(([files, root]) => {
+			return git.branches(root)
+			.then((branches) => {
+				return new checkoutDialog().activate(branches)
+			})
+			.then(([branch]) => {
+				return git.checkout(root, branch)
+				.then((result) => {
+					refreshAtom(root);
+					return result;
```

Comment:
maybe we should make it consistent with the other output than you could do another pull request with an option to show git result instead of generic response. I was also going to add an option to not show any notifications.